### PR TITLE
Removes deprecated environments 

### PR DESCRIPTION
**Pull request by nambrot**
Removes `dev` and `testnet` configuration from the rust and sdk configs

Keeps the `dev` config in `infra` as I imagine we would resurrect the dev environment at some point for agent development. Removed the testnet configs in infra as there should be no further use.

Scaled down the workloads on the cluster to 0 for now, will remove the namespace eventually if nothing breaks

### CODE REVIEW

**Comment by tkporter** [comment]
Only seeing this now 😵‍💫, happy to review again when conflicts are resolved

File: typescript/infra/config/environments/dev/index.ts [hunk 1]
```diff
@@ -1,18 +1,16 @@
-import { getMultiProviderFromGCP } from '../../../scripts/utils';
-import { CoreEnvironmentConfig } from '../../../src/config';
-import { Contexts } from '../../contexts';
+// import { getMultiProviderFromGCP } from '../../../scripts/utils';
```
Comment (tkporter) [inline typescript/infra/config/environments/dev/index.ts:1]: why not remove this and the rest of the `dev` directory?

**Review by tkporter — APPROVED** [review]
Could you rm the dev and testnet strings here as well https://github.com/abacus-network/abacus-monorepo/blob/main/rust/abacus-core/src/chain.rs#L123

File: typescript/infra/config/environments/dev/index.ts (hunk 1 above)
Comment (nambrot) [reply typescript/infra/config/environments/dev/index.ts]: I was originally thinking to keep it around if we will imminently create the dev environment but maybe thats too optimisitic, so just removed for now

**Pull request merged**